cgi/user.pl

```diff
@@ -433,6 +433,8 @@
 $template_data_ref->{type} = $type;
 $template_data_ref->{crm_contact_url} = get_contact_url($user_ref);
 $template_data_ref->{org_url} = ($user_ref->{org} ? "/cgi/org.pl?type=edit&orgid=$user_ref->{org}" : '');
+$template_data_ref->{localized_donate_url} = lang("donate_link");
```

Comment:
You don't have to pass translations to the template data, inside the template you can call lang() directly:

[% lang("donate_link") %]

For strings that are going to be between quotes (e.g. alt=".." tags), you can also escape them like this:

[% esq(lang("cta_donate")) %]   (esq means escape quotes)